include/taskolib/Sequence.h

```diff
@@ -69,6 +69,23 @@ namespace task {
  * // insert an action step at index 1
  * seq.insert(seq.begin() + 1, Step{Step::type_action});
  * \endcode
+ *
+ * ## Step setup script #
+ *
+ * The sequence contains a common setup script that is shared by all of its steps. It is
+ * called automatically before the execution of each step's script, just after executing
+ * the lua_step_setup function from the context. This step setup script can be set with
+ * set_step_setup_script(). It is typically used like a small library for defining common
+ * functions or constants.
+ *
+ * Global variables and functions defined in the setup script can be accessed in the
+ * script.
+ *
+ * The setup is only executed for Step types ACTION, IF, ELSEIF, or WHILE.
+ *
+ * \note The script is not validated and checked with a proper form. If a scripting error
+ * occurs during execution it will fail in the sequence and throw the appropriated Error
+ * exception.
```

Comment:
... and maybe we can make this more concise:
```
The setup is only executed for step types ACTION, IF, ELSEIF, or WHILE, and errors during its execution are handled like errors in the main script of the step.
```

Reply:
What are you missing as an external reader? I reduce it to ...

```
The setup is only executed for Step types ACTION, IF, ELSEIF, or WHILE.
```

But I don't now if this is sufficient. What do we need additionally?

Reply:
Nothing. I just find the `\note` paragraph a bit confusing and I think it is better to remove it. (What does "it will fail in the Sequence" mean? What is "the appropriated Error exception"?)

Reply:
Since the `step_setup` is move to `Context` I also remove the comment. BTW in `Context` the `\note` paragraph is now also removed. Anyway can you now check the description at `Context`?